common/utility.cpp

```diff
@@ -107,6 +107,7 @@ UString uniqueItemName(const UModelIndex & index)
         for (size_t j = 0; j < sizeof(table); j++) {
             if (name[i] == table[j]) {
                 name[i] = '_';
+                break;
```

Comment:
This code is very similar to fixFileName above. Can we reuse it?

Reply:
Fixed in 2926a9ff4625d10989bc2713517a0fe9f395abdf